tests/data/datacite-v4.1-full-example.json

```diff
@@ -0,0 +1,230 @@
+{
+  "identifier": {
+    "identifier": "10.5072/example-full",
+    "identifierType": "DOI"
+  },
+  "creators": [
+    {
+      "creatorName": {
+        "creatorName": "Miller, Elizabeth",
+        "nameType": "Personal"
+      },
+      "givenName": "Elizabeth",
+      "familyName": "Miller",
+      "nameIdentifiers": [
+        {
+          "nameIdentifier": "0000-0001-5000-0007",
+          "nameIdentifierScheme": "ORCID",
+          "schemeURI": "http://orcid.org/"
+        },
+        {
+          "nameIdentifier": "0100-0101-5100-0107",
+          "nameIdentifierScheme": "ORCID",
+          "schemeURI": "http://orcid.org/"
+        }
+      ],
+      "affiliations": ["DataCite", "CERN", "TIND"]
+    }
+  ],
+  "titles": [
+    {
+      "title": "Full DataCite XML Example",
+      "lang": "en-us"
+    },
+    {
+      "title": "Demonstration of DataCite Properties.",
+      "titleType": "Subtitle",
+      "lang": "en-us"
+    }
+  ],
+  "publisher": "DataCite",
+  "publicationYear": "2014",
+  "subjects": [
+    {
+      "subject": "000 computer science",
+      "subjectScheme": "dewey",
+      "valueURI": "https://cern.ch",
+      "schemeURI": "http://dewey.info/",
+      "lang": "en-us"
+    }
+  ],
+  "contributors": [
+    {
+      "contributorName": {
+        "contributorName": "Starr, Joan",
```

Comment:
because "nameType" was added, this is now a nested object instead of a string. I followed a patter from "identifier"